contracts/Proofs.sol

```diff
@@ -20,15 +19,15 @@ contract Proofs {
     downtime = __downtime;
   }
 
-  mapping(ProofId => bool) private ids;
-  mapping(ProofId => uint256) private starts;
-  mapping(EndId => uint256) private ends;
-  mapping(ProofId => EndId) private idEnds;
-  mapping(ProofId => uint256) private probabilities;
-  mapping(ProofId => uint256) private markers;
-  mapping(ProofId => uint256) private missed;
-  mapping(ProofId => mapping(uint256 => bool)) private received;
-  mapping(ProofId => mapping(uint256 => bool)) private missing;
+  mapping(SlotId => bool) private ids;
```

Comment:
Nitpick personal preference: could we call this `slotIds`?

Reply:
Yes, I'll change it.